<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BetterTimeSelection</class>
 <widget class="QMainWindow" name="BetterTimeSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1083</width>
         <height>598</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Date Selection</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCalendarWidget" name="calendarWidget">
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="nextMonth">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Terminal</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Next

Month</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="selectButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Terminal</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="prevMonth">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Terminal</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Previous

Month</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1083</width>
         <height>598</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Time Selection</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QMdiArea" name="mdiArea"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
